erigon-lib/state/domain_shared.go

```diff
@@ -1027,23 +1027,22 @@ func (sdc *SharedDomainsCommitmentContext) Account(plainKey []byte) (u *commitme
 	u.Flags |= commitment.BalanceUpdate
 	u.Balance.Set(&acc.Balance)
 
-	if ch := acc.CodeHash.Bytes(); len(ch) > 0 { // if code hash is not empty
+	if ch := acc.CodeHash.Bytes(); len(ch) > 0 {
 		u.Flags |= commitment.CodeUpdate
-		copy(u.CodeHash[:], ch)
+		copy(u.CodeHash[:], acc.CodeHash.Bytes())
```

Comment:
what difference?

Reply:
diff is below, so we do not fetch code and hash it if we see not empty code hash